### PR TITLE
Create ge_itdc

### PR DESCRIPTION
**Pull request by Dum4G**

### CODE REVIEW

File: streams/ge_itdc.m3u [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+#EXTINF:-1 tvg-id="",HD Life
+https://tbs01-edge11.itdc.ge/hdlife/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",365 дней ТВ 
+https://tbs01-edge11.itdc.ge/365day/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Авто Плюс
+https://tbs01-edge11.itdc.ge/autoplus/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Авто плюс
+http://109.200.135.130:8080/avto_plus/index.m3u8
+#EXTINF:-1 tvg-id="",Авто 24
+https://tbs01-edge11.itdc.ge/auto24/tracks-v1a1/mono.m3u8
```
Comment (freearhey) [inline streams/ge_itdc.m3u:113]: Duplicate.

File: streams/ge_itdc.m3u [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+#EXTINF:-1 tvg-id="",Viasat sport
+https://tbs01-edge11.Itdc.ge/viasatsport/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Mma-tv.com
+https://tbs01-edge11.Itdc.ge/m1/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",HD Life
+https://tbs01-edge11.itdc.ge/hdlife/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",365 дней ТВ 
+https://tbs01-edge11.itdc.ge/365day/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Авто Плюс
+https://tbs01-edge11.itdc.ge/autoplus/tracks-v1a1/mono.m3u8
```
Comment (freearhey) [inline streams/ge_itdc.m3u:109]: Duplicate.

File: streams/ge_itdc.m3u [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+#EXTINF:-1 tvg-id="",Уникум
+https://tbs01-edge11.Itdc.ge/detsky/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Viasat sport
+https://tbs01-edge11.Itdc.ge/viasatsport/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Mma-tv.com
+https://tbs01-edge11.Itdc.ge/m1/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",HD Life
+https://tbs01-edge11.itdc.ge/hdlife/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",365 дней ТВ 
+https://tbs01-edge11.itdc.ge/365day/tracks-v1a1/mono.m3u8
```
Comment (freearhey) [inline streams/ge_itdc.m3u:107]: Duplicate.

File: streams/ge_itdc.m3u [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+#EXTINF:-1 tvg-id="",Детский мир
+https://tbs01-edge11.Itdc.ge/detskymir/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Уникум
+https://tbs01-edge11.Itdc.ge/detsky/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Viasat sport
+https://tbs01-edge11.Itdc.ge/viasatsport/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",Mma-tv.com
+https://tbs01-edge11.Itdc.ge/m1/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="",HD Life
+https://tbs01-edge11.itdc.ge/hdlife/tracks-v1a1/mono.m3u8
```
Comment (freearhey) [inline streams/ge_itdc.m3u:105]: Duplicate.

**Review by freearhey — CHANGES_REQUESTED** [review]

File: streams/ge_itdc.m3u (hunk 3 above)
Comment (Dum4G) [reply streams/ge_itdc.m3u:99]: Fixed

File: streams/ge_itdc.m3u (hunk 2 above)
Comment (Dum4G) [reply streams/ge_itdc.m3u:101]: Fixed

File: streams/ge_itdc.m3u (hunk 1 above)
Comment (Dum4G) [reply streams/ge_itdc.m3u:103]: Fixed

File: streams/ge_itdc.m3u (hunk 4 above)
Comment (Dum4G) [reply streams/ge_itdc.m3u:97]: haven't found any duplicate for m1. Fixed

**Review by BellezaEmporium — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**